prow/config.yaml

```diff
@@ -14752,6 +14752,35 @@ periodics:
     - name: docker-graph
       emptyDir: {}
 
+- interval: 24h
+  name: ci-kubernetes-multicluster-ingress-test
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/bootstrap:v20180215-b2a89850e
```

Comment:
I see some jobs use bootstrap image and some use the kubetest image.
I assume kubetest is for e2e tests?

Reply:
note that bootstrap does not have go installed

Reply:
ah we need go :)
So I should change this to gcr.io/k8s-testimages/kubekins-e2e:v20180402-7b54c4ba6-master?

Reply:
make everything matches pull-kubernetes-multicluster-ingress-test?

Reply:
Done. For reference, pull-kubernetes-multicluster-ingress-test definition: https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L604